network/load-balancer/how-to/add-certificate.mdx

```diff
@@ -0,0 +1,162 @@
+---
+meta:
+  title: How to add a certificate to your Load Balancer
+  description: This page explains how to add a certificate to your Load Balancer
+content:
+  h1: How to add a certificate to your Load Balancer
+  paragraph: This page explains how to add a certificate to your Load Balancer
+dates:
+  validation: 2022-10-20
+  posted: 2022-10-20
+---
+
+You can add a certificate to your Load Balancer's frontend to enable the Load Balancer to encrypt/decrypt incoming traffic, necessary for [SSL bridging and SSL offloading configurations](/network/load-balancer/reference-content/configuring-load-balancer).
+
+<Message type="tip">
+
+Originally, certificates used the **S**ecure **S**ockets **L**ayer (SSL) cryptographic protocol, and are still often referred to simply as SSL certificates. However, it is now standard for such certificates not to use SSL at all, but rather its successor: **T**ransport **L**ayer **S**ecurity (TLS). [Transport Layer Security](https://en.wikipedia.org/wiki/Transport_Layer_Security) is a more up to date security protocol designed to increase privacy and data security for communications over the Internet. 
```

Comment:
Just an note here, the certificates do not use a protocol, it's the protocol that uses certificates. I would just "The initial encryption protocol used with HTTPS was SSL". or something like this.

Reply:
I'am OK with the modification.